fe/fe-core/src/main/java/com/starrocks/sql/optimizer/Utils.java

```diff
@@ -478,6 +478,22 @@ public static boolean canDoReplicatedJoin(OlapTable table, long selectedIndexId,
         return true;
     }
 
+    public static boolean containsEqualBinaryPredicate(ScalarOperator predicate) {
```

Comment:
What's the different with `isEqualBinaryPredicate`

Reply:
For expression `t1.a = t2.a and t1.b > t2.b`, the `isEqualBinaryPredicate` return false, but `isEqualBinaryPredicate` return true.